extension/framework/runner/environment.go

```diff
@@ -298,16 +313,10 @@ func (env *Environment) mockFunction(functionName string) {
 	})
 
 	env.setToOtto(functionName, "responses", []otto.Value{})
-	env.setToOtto(functionName, "Return", func(call otto.FunctionCall) otto.Value {
-		responses := env.getFromOtto(functionName, "responses").([]otto.Value)
-		if len(call.ArgumentList) != 1 {
-			call.Otto.Call("Fail", nil, "Return() should be called with exactly one argument")
-		}
-		responses = append(responses, call.ArgumentList[0])
-		env.setToOtto(functionName, "responses", responses)
+	env.setToOtto(functionName, "Return", getSpecifiedFunction(env,functionName,otto.FalseValue()))
 
-		return otto.NullValue()
-	})
+	env.setToOtto(functionName, "exceptions", []otto.Value{})
+	env.setToOtto(functionName, "Throw", getSpecifiedFunction(env,functionName,otto.TrueValue()))
```

Comment:
In one place there is a space after command and in other not. Fix the inconsistency.